src/test/javascript/spec/component/shared/metis/postings-footer/post-footer/post-footer.component.spec.ts

```diff
@@ -61,4 +92,81 @@ describe('PostFooterComponent', () => {
         const tags = getElements(fixture.debugElement, '.post-tag');
         expect(tags).toHaveLength(metisTags.length);
     });
+
+    it('should be initialized correctly for users that are at least tutors in course', () => {
+        component.posting = post;
+        component.posting.answers = unsortedAnswerArray;
+        metisServiceUserAuthorityStub.mockReturnValue(true);
+        component.ngOnInit();
+        expect(component.isAtLeastTutorInCourse).toBeTrue();
+        expect(component.createdAnswerPost.resolvesPost).toBeTrue();
+    });
+
+    it('should be initialized correctly for users that are not at least tutors in course', () => {
+        component.posting = post;
+        component.posting.answers = unsortedAnswerArray;
+        metisServiceUserAuthorityStub.mockReturnValue(false);
+        component.ngOnInit();
+        expect(component.isAtLeastTutorInCourse).toBeFalse();
+        expect(component.createdAnswerPost.resolvesPost).toBeFalse();
+    });
+
+    it('should contain an answer post', () => {
+        component.posting = post;
+        component.posting.answers = unsortedAnswerArray;
+        component.showAnswers = true;
+        fixture.detectChanges();
+        const answerPostComponent = fixture.debugElement.nativeElement.querySelector('jhi-answer-post');
+        expect(answerPostComponent).not.toBeNull();
+    });
+
+    it('should not contain an answer post', () => {
+        component.posting = post;
+        component.posting.answers = unsortedAnswerArray;
+        component.showAnswers = false;
+        fixture.detectChanges();
+        const answerPostComponent = fixture.debugElement.nativeElement.querySelector('jhi-answer-post');
+        expect(answerPostComponent).toBe(null);
```

Comment:
You're using both  ``toBe(null)`` and ``toBeNull()``. Please decide for one and use it consistently (would recommend the latter).

Reply:
Done ✔️